TypeScript with "moduleResolution": "bundler" can't import the npm package

**Steps to reproduce**
1. Create a TypeScript project with `"moduleResolution": "bundler"`
2. `yarn install imaginesdk` or `npm install imaginesdk`
3. Try to import `import { client } from 'imaginesdk'`

**Expected behavior**
`imaginesdk` module should be imported with all type definitions

**Actual behavior**
TypeScript build throws an error `ERROR(TypeScript)  Cannot find module 'imaginesdk' or its corresponding type declarations.`

**Additional context**
Node.js - v18.18.0
TypeScript - v5.2.2
Vite - v4.4.11

--- a/package.json
+++ b/package.json
@@ -32,7 +32,8 @@
   },
   "exports": {
     "require": "./dist/index.js",
-    "default": "./dist/index.modern.js"
+    "default": "./dist/index.modern.mjs",
+    "types": "./dist/index.d.ts"
   },
   "files": [
     "dist"

